src/main/java/com/openshift/internal/restclient/capability/resources/AbstractOpenShiftBinaryCapability.java

```diff
@@ -152,9 +154,11 @@ public final void start(final OpenShiftBinaryOption... options) {
 	}
 	
 	private void startProcess(final String location, final OpenShiftBinaryOption... options) {
-		String cmdLine = new StringBuilder(location).append(' ').append(buildArgs(Arrays.asList(options))).toString();
-		String[] args = StringUtils.split(cmdLine, " ");
+		List<String> args = new ArrayList<String>();
+		args.add("oc");
```

Comment:
dont think that we should hard code the binary name. Can't we just take the last segment of the cmd-line?